src/components/estimated-call/index.ts

```diff
@@ -0,0 +1,3 @@
+export {DepartureTime} from './DepartureTime';
+export {EstimatedCallInfo} from './EstimatedCallInfo';
+export {LineChip} from './LineChip';
```

Comment:
Not really, but I think it's a component that's fine to make available to the rest of the app. I guess we haven't talked about it before, but should we only export what's strictly necessary?

Reply:
Haven't really talked about it, but some thoughts from me:
- Stuff in src/components should reflect what are scoped as components in Figma. Even though they aren't in sync now, we can try to guess how it would look in Figma. Would the `LineChip` be a standalone component in Figma, or would it just be a part of the `EstimatedCallInfo` component?
- Components which are reusable through src/components are up to a little higher standard than components just scoped locally in another component-file. If the `LineChip` is considered good enough for a shared component, then it ok to export it.

Reply:
Makes sense! I'll leave it out for now.